admin/kernel-masterdata-service/src/test/java/io/mosip/kernel/masterdata/test/service/MasterDataServiceTest.java

```diff
@@ -3105,4 +3104,28 @@ public void validateRegCenterUpdateTest() {
 		Assert.assertEquals("KER-MSD-259", serviceErrors.get(4).getErrorCode());
 	}
 
+	@Test
+	public void getImmediateChildrenByLocCodeTest() {
+		Mockito.when(locationHierarchyRepository
+						.findLocationHierarchyByParentLocCode(Mockito.anyString(), Mockito.anyList()))
+				.thenReturn(locationHierarchies);
+		locationHierarchyService.getImmediateChildrenByLocCode("KAR", List.of("eng"));
```

Comment:
What are we asserting in this test case?

Reply:
yes corrected asserting Location code.